docker-compose.playwright.yaml

```diff
@@ -4,9 +4,9 @@
 #
 # This file comes from https://github.com/julienloizelet/ddev-playwright
 #
-version: '3.6'
 services:
   playwright:
+    # If on arm64 machine, edit to use mcr.microsoft.com/playwright:focal-arm64
```

Comment:
This is super important; I imagine you are on an amd64 machine. Not sure why MS doesn't publish multi-arch images.